EmployeeManagementSystem/DataAccessLayer/Entities/EmployeeEntity.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace DataAccessLayer.Entities
+{
+    public class EmployeeEntity
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
use IEntityTypeConfiguration to configure you db entities and relationships between them